modules/core/shared/src/main/scala/lucuma/core/enum/Band.scala

```diff
@@ -276,9 +276,39 @@ object Band {
         85.withRefinedUnit[Positive, Nanometer]
       )
 
+  /** @group Constructors */
+  case object GaiaG
+      extends BandWithDefaultUnits[VegaMagnitude, VegaMagnitudePerArcsec2](
+        "GaiaG",
+        "r",
+        "G",
+        Wavelength(690000),
+        720.withRefinedUnit[Positive, Nanometer] // 330-1050
+      )
+
+  /** @group Constructors */
+  case object GaiaGbp
+      extends BandWithDefaultUnits[VegaMagnitude, VegaMagnitudePerArcsec2](
+        "GaiaGbp",
+        "gbp",
+        "Gbp",
+        Wavelength(505000),
+        350.withRefinedUnit[Positive, Nanometer] // 330-680
+      )
+
+  /** @group Constructors */
+  case object GaiaGrp
+      extends BandWithDefaultUnits[VegaMagnitude, VegaMagnitudePerArcsec2](
+        "GaiaGrp",
+        "grp",
+        "Grp",
+        Wavelength(845000),
+        410.withRefinedUnit[Positive, Nanometer] // 640-1050
```

Comment:
Where does `410` come from?

Reply:
1050-640, that's what I understand as the width

Reply:
:+1: